program/build.rs

```diff
@@ -82,9 +150,9 @@ fn update_api() {
     {
         write_api(
```

Comment:
shouldn't above be similar cfg mapping, that contains
`all(feature = "gear", feature = "vara")`?